src/app/(nextjs_migration)/(guest)/breach-details/[breachName]/page.tsx

```diff
@@ -0,0 +1,368 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image, { StaticImageData } from "next/image";
+import BreachDetailScanImage from "../../../../../client/images/breach-detail-scan.svg";
+import "../../../../../client/css/partials/breachDetail.css";
```

Comment:
Question: I see `404` errors from `breachDetail.css` since for the current Monitor site icons are being imported via `background-image: url('/images/recommendation-icons/{iconName}.svg');` — not sure if we want to do anything with this right now?

Reply:
Oops, had to check those in: 4cbfa34ed

I just copied them over to `/public/`, though I'm not super happy about the filenames having no indication of it being transitional (like `/nextjs_migration/`), but :shrug:

Reply:
Solid catch @flozia!